src/test/kotlin/org/openrewrite/java/logging/slf4j/Slf4jBestPracticesTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2022 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.logging.slf4j
+
+import org.junit.jupiter.api.Test
+import org.openrewrite.Recipe
+import org.openrewrite.config.Environment
+import org.openrewrite.java.JavaParser
+import org.openrewrite.java.JavaRecipeTest
+
+class Slf4jBestPracticesTest : JavaRecipeTest {
```

Comment:
I'm not sure about the exact plans with regards to `org.openrewrite.java.JavaRecipeTest` versus `org.openrewrite.test.RewriteTest`. I've been using the latter lately, mostly from Java 17 tests, but I see both used still. Perhaps it would be good to note on both interfaces which to use when, and what the plans are for the future, especially for new contributions such as this one.